package.json

```diff
@@ -16,7 +16,7 @@
     "access": "public"
   },
   "engines": {
-    "node": "14 || 16 || 17 || 18 || 19"
```

Comment:
@achrinza any reason why you can't put `>=14` here so it will work for **21**, **22**, ... too?